tests/test_identify_utils.py

```diff
@@ -765,3 +766,73 @@ def test_edge_cases(self):
 
         with self.assertRaises(CausalGraphErrors.NodeDoesNotExistError):
             identify_markov_boundary(cg.skeleton, node='b')
+
+
+class TestIdentifyCollider(unittest.TestCase):
```

Comment:
No test for `TimeSeriesCausalGraph` whereas all the other identify utils have it. Think we need to explicitly test it.